### DISCUSSION TITLE
assertPageActionEnabled/assertPageActionExists not working for Edit/List resource pages

### DISCUSSION THREAD

**Discussion by nikspyratos** [filamentphp/filament#6101] [Help] [answered]
Heyo.

I've got a resource where the delete action is available in the main resource table (as defined in the Resource class) and in the Edit page (as defined in the EditResource page class).

In my tests I'm trying to do:
```
Livewire::actingAs($this->user)
            ->test(ExampleResource::class, ['record' => $this->example->getRouteKey()])
            ->assertPageActionExists(DeleteAction::class);
```
And it always fails:
```
Failed asserting that an action with name [Filament\Pages\Actions\DeleteAction] exists on the [App\Filament\Resources\ExampleResource\Pages\EditExample] page.

```

At first I thought it was just with the ListExample page since the row-level actions are defined in the main ExampleResource (which is not a testable livewire component as far as I can tell), but this happens for the Edit page as well.

I don't have any special permission/authorisation checks for this action, and running `callPageAction`  runs successfully.

Not sure if I'm missing something here or if there's any special kind of usage in a test? The docs all being in Pest does mean they're not always equivalent for traditional PHPUnit style usage.

**Comment by danharrin** [comment, marked as answer]
Please make sure you're on the latest v2 version because I thought I fixed this the other day

**Reply by nikspyratos** [reply]
Will have a look and report back thanks!

**Reply by nikspyratos** [reply]
Latest update solved it thanks!
